template/shell.nix.j2

```diff
@@ -0,0 +1,44 @@
+let
+  self = import ./. {};
+  inherit (self) sources pkgs;
+
+  beku = pkgs.callPackage (sources."beku.py" + "/beku.nix") {};
+  cargoDependencySetOfCrate = crate: [ crate ] ++ pkgs.lib.concatMap cargoDependencySetOfCrate (crate.dependencies ++ crate.buildDependencies);
+  cargoDependencySet = pkgs.lib.unique (pkgs.lib.flatten (pkgs.lib.mapAttrsToList (crateName: crate: cargoDependencySetOfCrate crate.build) self.cargo.workspaceMembers));
+in pkgs.mkShell rec {
+  name = "{[ operator.name }]";
```

Comment:
This doesn't need to use Jinja, it can be read from nix/meta.json just like how default.nix does it.

Reply:
Ohh yes, good point. now we get syntax highlighting back :D

Reply:
Done in 47fc626